invenio_vocabularies/services/tasks.py

```diff
@@ -26,3 +27,32 @@ def process_datastream(config):
         if result.errors:
             for err in result.errors:
                 current_app.logger.error(err)
+
+
+@shared_task(bind=True)
+def import_funders(self):
+    """Import the funders vocabulary.
```

Comment:
This task can be an unbound task now since we are doing state management externally.

Reply:
Done, changed the method signature to:

```python
@shared_task()
def import_funders():
```